compiler/packages/react-compiler-healthcheck/src/checks/reactCompiler.ts

```diff
@@ -121,5 +121,36 @@ export default {
         `Successfully compiled ${SucessfulCompilation.length} out of ${totalComponents} components.`
       )
     );
+
+    if (verbose) {
+      for (const compilation of [...SucessfulCompilation, ...ActionableFailures, ...OtherFailures]) {
+        const filename = compilation.fnLoc?.filename;
+
+        if (compilation.kind === "CompileSuccess") {
+          const name = compilation.fnName;
+          const isHook = name?.startsWith('use');
+
+          if (name) {
+            console.log(
+              chalk.green(
+                `Successfully compiled ${isHook ? "hook" : "component" } [${name}](${filename})`
+              )
+            );
+          } else {
+            console.log(chalk.green(`Successfully compiled ${compilation.fnLoc?.filename}`));
+          }
+        }
+
+        if (compilation.kind === "CompileError") {
```

Comment:
Certain errors have severity `ErrorSeverity.Todo`. I don't think it's worth making it all red, but I don't think we should skip this information either. So perhaps make it a different color or something?